documentation/src/main/docs/index.adoc

```diff
@@ -19,9 +19,13 @@ ifdef::basebackend-html[toc::[]]
 :sectnumlevels: 4
 
 include::intro.adoc[]
+
 include::getting-started.adoc[]
+
 include::philosophy.adoc[]
+
 //include::sections/converters.adoc[]
+
 //include::sections/decision-table.adoc[]
-include::faq.adoc[]
 
+include::faq.adoc[]
```

Comment:
If you take a look at the online doc (https://smallrye.io/smallrye-mutiny/), you'll see that all top-level titles are currently broken. I'm not sure the changes I made in this file are the good way to fix that though. I'm not familiar with the mutiny build and I couldn't generate the doc locally to verify the fix.